### DISCUSSION TITLE
Is it possible to change component default values globally?

### DISCUSSION THREAD

**Discussion by HitomiTenshi** [heroui-inc/heroui#3777] [Q&A] [answered]
Hi, I want my tooltips to always have a `closeDelay={0}` and the color to always be `color="foreground"`. Right now I have to enter these 2 settings for every tooltip in my React app, that is very cumbersome. Is there a way to set these values globally?

**Comment by wingkwong** [comment, marked as answer]
No. In this case, you can simply create a custom Tooltip with `closeDelay={0}` and `color="foreground"` extending NextUI Tooltip.

**Reply by HitomiTenshi** [reply]
I solved it like this, in case someone comes across this issue:

```typescript
import {Tooltip as NextUITooltip, TooltipProps} from '@nextui-org/tooltip'

export default function Tooltip(props: TooltipProps) {
  return <NextUITooltip closeDelay={0} color="foreground" {...props} />
}
```

In this way, `closeDelay` and `color` have a new user-defined default value, but can be overwritten if they are defined.
